From 3a980e0cf5a2339078501b75caf46d4b728b0c27 Mon Sep 17 00:00:00 2001
From: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Date: Mon, 27 Aug 2018 12:58:00 +0900
Subject: [PATCH 524/909] arm64: dts: r8a77995-draak: Set preferred resolution

commit 2568909cb5c7c59903cca3e6a11427f80596b201 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

On the Draak board (R8A77990), add option to set the initial
resolution.

Signed-off-by: Koji Matsuoka <koji.matsuoka.xm@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77995-draak.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77995-draak.dts b/arch/arm64/boot/dts/renesas/r8a77995-draak.dts
index 4e34a9ac0563..5d22f1206575 100644
--- a/arch/arm64/boot/dts/renesas/r8a77995-draak.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77995-draak.dts
@@ -29,6 +29,9 @@
 
 		no-use-ddc;
 
+		width = <800>;
+		height = <600>;
+
 		port {
 			vga_in: endpoint {
 				remote-endpoint = <&adv7123_out>;
-- 
2.17.1

